README.md

```diff
@@ -1,18 +1,23 @@
 ## Welcome to dotnet sdk
 
-Core functionality needed to create .NET Core projects, that is shared between VisualStudio and CLI.
+This repo contains core functionality needed to create .NET Core projects, that is shared between VisualStudio and CLI.
 
 * MSBuild tasks can be found under [/src/Tasks/Microsoft.NET.Build.Tasks/](src/Tasks/Microsoft.NET.Build.Tasks).
 
-Please refer to [dotnet/cli](https://github.com/dotnet/cli) for .NET Core CLI source code, and to [dotnet/project-system](https://github.com/dotnet/project-system) for the project system work that is specific to Visual Studio 2017.
+Please refer to [dotnet/cli](https://github.com/dotnet/cli) for .NET Core CLI source code, and to [dotnet/project-system](https://github.com/dotnet/project-system) for the project system work that is specific to Visual Studio.
 
-## Build Status
+## Build status
 
 |Windows x64 |
 |:------:|
 |[![](https://dev.azure.com/dnceng/internal/_apis/build/status/dotnet/sdk/DotNet-Core-Sdk%203.0%20(Windows)%20(YAML)%20(Official))](https://dev.azure.com/dnceng/internal/_build?definitionId=140)|
 
+## Testing a local build
+
+To test your locally built SDK, run `eng\dogfood.cmd` after a building. That script starts a new Powershell with the environment configured to redirect SDK resolution to your build. Then run `& devenv.exe`.
```

Comment:
We should also doc that msbuild and dotnet build will use your changes as well.

Reply:
Outside of VS you mean?

Reply:
Yeah, I mean you can do command line testing of your bits this way.

Reply:
Nice. Updated.